src/main/asciidoc/index.adoc

```diff
@@ -140,3 +140,36 @@ You can change the control, e.g you can send a response immediately to the user-
 ----
 {@link examples.HttpProxyExamples#immediateResponse}
 ----
+
+==== Header interceptor
+
+You can apply header interceptors to change headers from the request and response with common operations:
+
+[source,java]
+----
+{@link examples.HttpProxyExamples#headerInterceptorFilter}
+----
+
+Or apply a callback:
+
+[source,java]
+----
+{@link examples.HttpProxyExamples#headerInterceptorCallback}
+----
```

Comment:
Instead of giving a second example, I think it would be more valuable to invite the reader to consult the doc of `HeadersInterceptor`.


```suggestion
Check out the {@link io.vertx.httpproxy.interceptors.HeadersInterceptor} class for details about the available methods.
```

Reply:
When the documentation is generated and published to the website, `{@link io.vertx.httpproxy.interceptors.HeadersInterceptor}` becomes an hyper-text link to the corresponding Javadoc. We do this in several places in the documentation.

For example: https://vertx.io/docs/vertx-web-client/java/#_predefined_expectations (end of the section)